mods/cnc/chrome/music.yaml

```diff
@@ -139,7 +139,7 @@ Container@MUSIC_PANEL:
 							Ticks: 7
 				Label@TIME_LABEL:
 					X: (PARENT_RIGHT - WIDTH) / 2
-					Y: 315
+					Y: 316
```

Comment:
If we move this down one px (or the Shuffle / Loop checkboxes up by one px) then they'll share a common baseline, which would IMO look better.